### DISCUSSION TITLE
Invalid due date?

### DISCUSSION THREAD

**Discussion by neville-nazerane** [dotnet/csharplang#2199] [Q&A] [answered]
I think (and hope) the due date is an accident. It mentions the year 2080.

**Comment by yaakov-h** [comment]
Context/link?

**Comment by neville-nazerane** [comment]
https://github.com/dotnet/csharplang/milestone/8

I thought the milestone was linked.

**Comment by yaakov-h** [comment, marked as answer]
It looks like somebody has just repurposed the last two digits of the year. 

<img width="227" alt="image" src="https://user-images.githubusercontent.com/426009/52254044-7d294780-295f-11e9-9574-7cf321ea9332.png">

**Comment by neville-nazerane** [comment]
Either that was intentional or an AI like excel's.

**Comment by ufcpp** [comment]
IIRC, it's intentional. C# team doesn't want to declare proper due date.

**Comment by yaakov-h** [comment]
True, but they also didn’t have to specify one at all. The field is optional.

**Comment by YairHalberstadt** [comment]
I'm afraid it's correct. C# 8.0 is planned for early 2080 all going well.

**Comment by neville-nazerane** [comment]
Ok, I just had to make sure. And I am afraid too.

**Comment by alrz** [comment]
>True, but they also didn’t have to specify one at all. 

But if it's there, then no one needs to ask..

**Comment by MgSam** [comment]
> > True, but they also didn’t have to specify one at all.
> 
> But if it's there, then no one needs to ask..

Doesn't look like that plan worked out.

**Comment by alrz** [comment]
think the team was going to match langversion with the year (https://github.com/dotnet/csharplang/issues/2199#issuecomment-460513973) but just got ahead of schedule.
